### PR TITLE
low_cpu_mem refinement

### PR DESCRIPTION
**Pull request by n1ck-guo**

### CODE REVIEW

File: auto_round/autoround.py [hunk 1]
```diff
@@ -86,6 +86,7 @@ class AutoRound(object):
         lr (float): The learning rate (default is None, will be set to 1.0/iters).
         minmax_lr (float): The learning rate for min-max tuning (default is None, it will be set to lr automatically).
         low_gpu_mem_usage (bool): Whether to use low GPU memory (default is True).
+        low_cpu_mem_usage (bool): Whether to use low CPU memory (default is False).
```
Comment (wenhuach21) [inline auto_round/autoround.py:89]: rename layer_wise folder to low_cpu_xxx

File: examples/language-modeling/main.py [hunk 2]
```diff
@@ -117,7 +117,10 @@
                         help="quant_lm_head")
 
     parser.add_argument("--low_cpu_mem_mode", default=0, type=int,
-                        help="choose low cpu memory mode, 1 for block-wise, 2 for layer-wise, others means not use low cpu memory.")
+                        help="Choose which low cpu memory mode to use. Can significantly reduse cpu memory footprint but cost more time."
+                        "1 means choose block-wise mode, load the weights of each block from disk when tuning and release the memory of the block after tuning."
+                        "2 means chosse layer-wise mode, load the weights of each layer from disk when tuning, minimum memory consumption and also slowest runing speed."
```
Comment (wenhuach21) [inline examples/language-modeling/main.py]: set a variable for "layer_wise_tmp"

**Comment by chensuyue** [comment]
Does this target v0.3?

**Comment by wenhuach21** [comment]
> Does this target v0.3?

Yes, if possible. This PR only changes comments or moves some folders.

File: examples/language-modeling/main.py (hunk 2 above)
Comment (n1ck-guo) [reply examples/language-modeling/main.py]: done

File: auto_round/autoround.py (hunk 1 above)
Comment (n1ck-guo) [reply auto_round/autoround.py:89]: done

File: examples/language-modeling/main.py (hunk 2 above)
Comment (n1ck-guo) [reply examples/language-modeling/main.py]: done

**Review by wenhuach21 — APPROVED** [review]

**Pull request merged**